receiver/httpcheckreceiver/factory_test.go

```diff
@@ -43,9 +43,13 @@ func TestNewFactory(t *testing.T) {
 						CollectionInterval: 60 * time.Second,
 						InitialDelay:       time.Second,
 					},
-					HTTPClientSettings:   httpSettings,
 					MetricsBuilderConfig: metadata.DefaultMetricsBuilderConfig(),
-					Method:               "GET",
+					Targets: []*targetConfig{
```

Comment:
Shouldn't the default config have an empty list of targets?

Reply:
When I initially worked on this, there was a default endpoint. I thought that that behaviour should be kept as is. Looks like it remove now. I can update this.